src/main/steps/existing-application/post.test.ts

```diff
@@ -36,52 +43,61 @@ describe('ExistingApplicationPostController', () => {
     expect(res.redirect).toBeCalledWith(SAVE_AND_SIGN_OUT);
   });
 
-  test('Should have no errors and redirect to the access code page', async () => {
+  test.each([
+    [ApplicationType.JOINT_APPLICATION, undefined, undefined],
```

Comment:
dont think you need to set the undefined parts if both test are expecting it.

Reply:
agreed, will update